### PR TITLE
Update UCNF README

### PR DESCRIPTION
**Pull request by nickolaev**
Signed-off-by: Nikolay Nikolaev <nnikolay@vmware.com>

### CODE REVIEW

File: examples/universal-cnf/README.md [hunk 1]
```diff
@@ -1,7 +1,126 @@
...
+
+UCNF configuration is based on events and actions. The supported events are generated on initialization and upon Endpoint connection request. The available actions are: command line execution, Client and dataplane configuration.
+
+Here is a more detailed explanation of the supported event types:
+ * The initialization event is the first event generated when UCNF starts. The actions associated with it will be executed before anything else in the system. This can typically be used to execute any pre-configured Clients or container wide commands.
+
+ * The Endpoint connection request event is generated when a Client requests a connection from the UCNF announced Endpoint.
+
+The actions which UCNF supports are as follows:
```
Comment (ivanayov) [inline examples/universal-cnf/README.md:57]: Can be a bit confusing and not clear how commands, client and dataplane are associated with actions.

File: examples/universal-cnf/README.md [hunk 2]
```diff
@@ -1,7 +1,126 @@
 # The Universal CNF
 
-Universal CNF is an attempt to implement a swiss-army knife container, which implements basic networking functions useful for demoing NSM. It is currently heavily based in VPP and vppagent.
+Universal CNF (UCNF) is an attempt to implement a swiss-army knife container, which implements basic networking functions useful for demoing NSM. It is currently heavily based on [VPP](https://github.com/FDio/vpp) and [vppagent](https://github.com/ligato/vpp-agent).
 
 ## Configuration
 
-TBD
\ No newline at end of file
+UCNF can be deployed and configured to implement various cloud-native netowrking scenarios. For this, it exposes a configuration file interface, where the solution integrator can program it.
```
Comment (ivanayov) [inline examples/universal-cnf/README.md]: I think "For that" is more correct

**Comment by ivanayov** [comment]
Looks very good and clear to follow and understand. Two comments only.

Some diagram for the given examples would give more visual understanding, but it's also ok and clear this way.

File: examples/universal-cnf/README.md (hunk 2 above)
Comment (rdimitrov) [reply examples/universal-cnf/README.md]: Great write up! Found one small typo in "networking", but other than that it looks great 👍

**Pull request merged**